pkg/imagefilter/formatter.go

```diff
@@ -5,17 +5,30 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"sort"
+	"strings"
 )
 
 // OutputFormat contains the valid output formats for formatting results
 type OutputFormat string
 
 const (
-	OutputFormatDefault OutputFormat = ""
-	OutputFormatText    OutputFormat = "text"
-	OutputFormatJSON    OutputFormat = "json"
+	OutputFormatDefault   OutputFormat = ""
+	OutputFormatText      OutputFormat = "text"
+	OutputFormatJSON      OutputFormat = "json"
+	OutputFormatTextShell OutputFormat = "shell"
+	OutputFormatTextShort OutputFormat = "short"
 )
 
+func GetSupportedOutputFormats() []string {
```

Comment:
The convention in go is to not use the `Get` prefix (c.f. https://go.dev/doc/effective_go#Getters) so maybe just `SupportedOuptutFormats()` ?

Reply:
Thanks for you patience while I'm trying to learn go :-D